apis/tools/env/5.0/wazuh-agent/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM ubuntu:24.04 AS builder
+
+# Install dependencies
+RUN apt-get update && apt-get install -y build-essential cmake make gcc git zip curl tar ninja-build pkg-config wget \
+    gnupg lsb-release software-properties-common libbz2-dev iputils-ping
+
+RUN wget https://apt.llvm.org/llvm.sh && chmod +x llvm.sh && ./llvm.sh 18
+
+RUN apt-get update && apt-get install -y clang-tidy-18 autopoint libtool zlib1g-dev libgcrypt20-dev libmagic-dev \
+    libpopt-dev libmagic-dev libsqlite3-dev liblua5.4-dev gettext libarchive-dev libxml2
+
+# Clone repository and initialize submodules
+RUN git clone https://github.com/wazuh/wazuh-agent.git --single-branch --depth=1
+
+WORKDIR /wazuh-agent
+
+RUN git submodule update --init --recursive
+
+# Compile the binary
+RUN cmake /wazuh-agent/src -B /wazuh-agent/build
+
+RUN cmake --build /wazuh-agent/build
+
+FROM ubuntu:24.04 AS agent
+
+RUN apt update && apt install -y iputils-ping curl
```

Comment:
This is on purpose, because the `agent` image doesn't contain anything from the `builder` image (apart from what's copied) to make it as lightweight as possible.

Moving this to the previous stage would mean we don't have them in this one, unless we copy those specific utilities (which would be the same as now).